.gitignore

```diff
@@ -17,4 +17,7 @@ coverage
 mytestmodel
 
 # OS generated files
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+.venv
+**/**/dist/
```

Comment:
.venv is already ignored - line 2. Don't ignore `**/**/dist/` for now - as discussed in the previous meeting, we agreed to push the build artifacts to facilitate testing purposes for the time being

Reply:
done